tardis/plasma/properties/nlte_excitation_data.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+
+class NLTEExcitationData(object):
+    def __init__(self, atom_data_lines, nlte_excitation_species):
+        self.atom_data_lines = atom_data_lines
+        self.lines = atom_data_lines.reset_index()
+        self.nlte_excitation_species = nlte_excitation_species
+
+        if nlte_excitation_species:
```

Comment:
perhaps nlte_excitation_species should be a kwarg set to None/False?

Reply:
It's default is empty either way, and I will be calling this function only when nlte_excitation_species is not empty